rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -82,3 +82,8 @@ nova_ceilometer_enabled: False
 swift_ceilometer_enabled: False
 keystone_ceilometer_enabled: False
 tempest_service_available_ceilometer: False
+
+neutron_git_repo: "https://git.openstack.org/openstack/neutron"
+neutron_git_install_branch: "b80fa23ac0b292db84df84855141eb16a4e0fec0"
+neutron_git_dest: "/opt/neutron_{{ neutron_git_install_branch | replace('/', '_') }}"
```

Comment:
So we need the `neutron_git_repo` and `neutron_git_dest` vars redefined in order to update the neutron branch? That seems unwieldy, but fixing that behavior isn't in scope, more looking to verify that it's necessary.